MODULE.bazel

```diff
@@ -0,0 +1,22 @@
+module(
+    name = "com_google_googleapis",
+    version = "0.0.1",
```

Comment:
You can leave this at `0.0.0` and rely on the Publish to the BCR plugin to patch in the real version number when a tag is created. Well, if no tags are created we will need to release this manually, e.g. using Go pseudo-versions.